src/cockpit/389-console/src/lib/database/localPwp.jsx

```diff
@@ -2804,19 +2799,13 @@ export class LocalPwPolicy extends React.Component {
                                             }}
                                             aria-label="FormSelect Input"
                                         >
-                                            <FormSelectOption key="0" value="PBKDF2_SHA256" label="PBKDF2_SHA256" />
-                                            <FormSelectOption key="1" value="SSHA512" label="SSHA512" />
-                                            <FormSelectOption key="2" value="SSHA384" label="SSHA384" />
-                                            <FormSelectOption key="3" value="SSHA256" label="SSHA256" />
-                                            <FormSelectOption key="4" value="SSHA" label="SSHA" />
-                                            <FormSelectOption key="5" value="MD5" label="MD5" />
-                                            <FormSelectOption key="6" value="SMD5" label="SMD5" />
-                                            <FormSelectOption key="7" value="CRYPT-MD5" label="CRYPT-MD5" />
-                                            <FormSelectOption key="8" value="CRYPT-SHA512" label="CRYPT-SHA512" />
-                                            <FormSelectOption key="9" value="CRYPT-SHA256" label="CRYPT-SHA256" />
-                                            <FormSelectOption key="10" value="CRYPT" label="CRYPT" />
-                                            <FormSelectOption key="11" value="GOST_YESCRYPT" label="GOST_YESCRYPT" />
-                                            <FormSelectOption key="12" value="CLEAR" label="CLEAR" />
+                                            {this.props.pwdStorageSchemes.map((option, index) => (
```

Comment:
For some reason, the select doesn't work for the Local PW policy... But works for the Global Pwp.

[Screencast from 02-13-2023 07:55:56 PM.webm](https://user-images.githubusercontent.com/5634892/218635606-63caa77c-2b6a-4d81-b512-3b196fee6cac.webm)